src/index.ts

```diff
@@ -0,0 +1,40 @@
+import { write_int } from "./patch_memory"
```

Comment:
looks very bad.. I expected from that lang to have import-all feature by default, and not expect devs to add everything to their `__all__` export